### PR TITLE
[sonic-mux-cable.yang]: yang model for mux_cable table.

### PR DESCRIPTION
**Pull request by praveen-li**
Changes:
-- yang model for mux_cable table.
-- yang model tests.
-- fix setup.py to have sorted list of yang files

Signed-off-by: Praveen Chaudhary <pchaudhary@linkedin.com>

Note: Do not merge it. Need a dependent PR.

#### Why I did it

yang model for mux_cable table.

#### How I did it

-- yang model for mux_cable table.
-- yang model tests.
-- fix setup.py to have sorted list of yang files

#### How to verify it

-- yang model build time tests.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

### CODE REVIEW

**Comment by praveen-li** [comment]
@prsunny Kindly review. Let me know if any more conditions are needed in yang model for mux cable config.

@zhangyanzhao

File: src/sonic-yang-models/yang-models/sonic-mux-cable.yang [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        description "Initial version";
+    }
+
+    container sonic-mux-cable {
+
+        container MUX_CABLE {
+
+            description "MUX_CABLE part of config_db.json";
+
+            leaf PEER_TOR {
```
Comment (prsunny) [inline src/sonic-yang-models/yang-models/sonic-mux-cable.yang:29]: Peer_tor is not part of the MUX_CABLE table but part of PEER_SWITCH table. Please remove from here

**Review by prsunny — COMMENTED** [review]
thanks @praveen-li for the PR. Few comments

File: src/sonic-yang-models/yang-models/sonic-mux-cable.yang (hunk 1 above)
Comment (praveen-li) [reply src/sonic-yang-models/yang-models/sonic-mux-cable.yang]: Thx Prince. 
Kindly also suggest if any condition is missing from yang models. For example, say, if SERVER ipv4 address should be unique across PORTS.

Comment (prsunny) [reply src/sonic-yang-models/yang-models/sonic-mux-cable.yang]: Yes, thats an implicit assumption. But  agree with you.. we can make it part of the yang.

**Comment by zhangyanzhao** [comment]
Thanks Praveen. Looks like there is dup with https://github.com/sonic-net/sonic-buildimage/pull/11797, some communication gap on my side. Sorry for the inconvenience.

**Pull request closed without merging**